doc/developer/buildchain.md

```diff
@@ -89,3 +89,40 @@ OPTION( RADIUM_SKIP_${NAME_UPPER}_EXTERNAL "[addExternalFolder] Skip updating ${
 ~~~
 where `$NAME_UPPER` is `CORE`, `ENGINE`, `GUIBASE`, `PLUGINBASE` or `IO`.
 \note The option `RADIUM_SKIP_${NAME_UPPER}_EXTERNAL` is generated only if `RADIUM_GENERATE_LIB_${NAME_UPPER}` is `ON`.
+
+## User provided external dependencies
+
+You can  use your own installation of a local dependency instead of letting cmake fetch and comiple it.
+To this end, just provide the corresponding '*_DIR' to cmake at configuration time (with '-D' option, configuration file or toolchain file, please refer to cmake documentation).
+
+Currently supported:
+
+ *  assimp\_DIR
+ *  tinyply\_DIR
+ *  glm\_DIR
+ *  glbinding\_DIR
+ *  globjects\_DIR
+ *  stb\_DIR
+ *  Eigen3\_DIR
+ *  OpenMesh\_DIR
+ *  cpplocate\_DIR
+
+Radium in compiled and tested with specific version of dependencies, as given in the external's folder CMakeLists.txt and state here for the record
+
```

Comment:
Could’t we use snippets from the cmake files directly ?

Reply:
not sure what do you mean ? including the cmake files here ? actually I've done a python script to generate this text from the cmake file.

Reply:
Ok fine then 👍

Reply:
While this is easy for code files, I am not sure doxygen will be able to produce similar results for cmake files.

Reply:
well, feel free to try, the script is below (see scripts/list_dep.py)
if you find a easy way to obtain something similar using snippets in cmakelists without redundancies.

Reply:
OK, just stay like this. Even if doxygen handle cmake scripts quite well, it is not so crucial to add cmake snipets. The script does the job well.